Packs/FireEye-Detection-on-Demand/pack_metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "FireEye Detection on Demand",
+    "description": "Detonate files, hashes, and URLs using FireEye Detection on Demand",
+    "support": "developer",
+    "currentVersion": "1.0.0",
+    "author": "FireEye Inc.",
+    "url": "https://fireeye.dev/docs/detection-on-demand",
+    "email": "developers@fireeye.com",
+    "created": "2020-10-27T14:49:02Z",
+    "categories": [
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [
+        "fireeye",
+        "detection",
+        "demand",
+        "detonate",
+        "malware",
+        "analysis",
+        "lookup",
+        "hash",
+        "file",
+        "url"
+    ],
+    "useCases": [],
+    "keywords": []
+}
```

Comment:
Please also add your gihub user in the following format. That way you'll be notified in any change of the pack.
```
    "githubUser": [
        "<GitHub_User>"
    ]
```